Modules/TOF/include/TOF/TOFMatchedTracks.h

```diff
@@ -72,34 +77,41 @@ class TOFMatchedTracks final : public TaskInterface
   std::shared_ptr<o2::globaltracking::DataRequest> mDataRequest;
   o2::globaltracking::RecoContainer mRecoCont;
   GID::mask_t mSrc = GID::getSourcesMask("ITS-TPC");
-  GID::mask_t mAllowedSources = GID::getSourcesMask("TPC,ITS-TPC,TPC-TOF,ITS-TPC-TOF");
+  GID::mask_t mAllowedSources = GID::getSourcesMask("TPC,ITS-TPC,TPC-TOF,ITS-TPC-TOF,TPC-TRD,ITS-TPC-TRD-TOF,ITS-TPC-TRD-TOF");
```

Comment:
You have twice ITS-TPC-TRD-TOF but not TPC-TRD-TF

Reply:
A typo while I tried to add TRD back, thanks for spotting (there was also TPC-TRD missing)